pkg/debug/scripts/collect-info.sh

```diff
@@ -303,6 +303,23 @@ collect_kube_info()
     fi
 }
 
+collect_longhorn_info()
+{
+    type=$(cat /run/eve-hv-type)
+    if [ "$type" != "kubevirt" ]; then
+        return
+    fi
+    echo "- Collecting Longhorn specific info"
+    {
+        echo "  - longhorn support bundle "
+        # Longhorn issue reports have seen hangs generating this data 
+        # when nodes are down, or due to longhorn bugs.  
+        # Give up after 5min, and allow remaining system data to be collected.  
+        timeout 300s eve exec kube /usr/bin/longhorn-generate-support-bundle.sh
+        cp /persist/newlog/kube/longhornsupportbundle* "$DIR"/
```

Comment:
Why do we need this copy ?  I though collect info will collect everything under /persist/newlog/kube

Reply:
You're right, I'll pull it out

Reply:
@zedi-pramodh latest commit removed it